shared/chat/conversation/index.desktop.js

```diff
@@ -175,7 +176,11 @@ export default branch(
   renderComponent(NoConversation),
   branch(
     (props: Props) => !!props.rekeyInfo,
-    renderComponent(ParticipantRekey),
+    branch(
```

Comment:
Yeah this thing is creaky. Likely better to make a new component in this file to express this. I don't think the connected component should know about these rendering details though. Luckily this is the last of these branching paths so